examples/c/simple_dae_model/export_simple_dae_model.m

```diff
@@ -0,0 +1,61 @@
+function [ model ] = export_simple_dae_model()
+    %% this function generates an implicit ODE / index-1 DAE model,
+    % which consists of a CasADi expression f_impl_expr
+    % that depends on the symbolic CasADi variables x, xdot, u, z,
+    % and a model name, which will be used as a prefix for generated C
+    % functions later on;
+    
+    %% CasADi
+    import casadi.*
+    if CasadiMeta.version()=='3.4.0'
+        % casadi 3.4
+        casadi_opts = struct('mex', false, 'casadi_int', 'int', 'casadi_real', 'double');
+    else
+        % old casadi versions
+        error('Please download and install Casadi 3.4.0 to ensure compatibility with acados')
+    end
+    model_name_prefix = 'inv_pendulum';
```

Comment:
I made a PR to your PR, such that the model is generated using the functions in `external_function_generation`.